binance-dex/adapter.js

```diff
@@ -0,0 +1,58 @@
+const { Requester, Validator } = require('@chainlink/external-adapter')
+const apiEndpoint = process.env.API_ENDPOINT || 'dex-asiapacific'
+
+const customError = (data) => {
```

Comment:
```suggestion
const isCustomError = (data) => {
```

Reply:
I think we can get rid of this function all together actually, since we have no good point of checking for errors with this.